### PR TITLE
Properly configure tzdata for TF GPU Kernel image

### PR DESCRIPTION
**Pull request by lresende**

### CODE REVIEW

File: etc/docker/kernel-tf-gpu-py/Dockerfile [hunk 1]
```diff
@@ -9,11 +9,12 @@ RUN apt-get update && apt-get install -yq \
     libxrender1 \
     netcat \
     python3-dev \
-    tzdata \
-    unzip \
-    && rm -rf /var/lib/apt/lists/*
-
-RUN pip install future pycryptodomex
+    unzip && \
+    ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
```
Comment (kevin-bates) [inline etc/docker/kernel-tf-gpu-py/Dockerfile:13]: Could you explain why this is necessary?  It doesn't feel right as it seems to pin the TZ to EST.  A brief comment would be sufficient.

Comment (lresende) [reply etc/docker/kernel-tf-gpu-py/Dockerfile:13]: Newer version of tzdata now requires some setup in order to be properly installed, and this is just `a choice` of timezone. Another question could be, do we need to install data ?

Comment (lresende) [reply etc/docker/kernel-tf-gpu-py/Dockerfile]: I found a way to disable the requirement to set a timezone while installing tzdata using non-interactive env var, should have the same behavior as before now.

**Review by kevin-bates — APPROVED** [review]
LGTM - thanks Luciano!

**Pull request merged**